docs/04-create-addons.md

```diff
@@ -0,0 +1,116 @@
+        ├── {assets}                         # Files with documentation and assets
+        └── ....
+```
+
+> **NOTE:** All file names in an addon repository are case-sensitive.
+
+For details about particular files, read the following sections.
+
+## meta.yaml file
+
+The `meta.yaml` file contains information about the addon. Define the following fields to create a service object which complies with the [Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker/blob/v2.14/spec.md#service-object).
+
+|      Field Name     | Required |                   Description             |
+|-------------------|:--------:|----------------------------------------------|
+|         **name**        |   Yes   | The name of the addon.  |
+|       **version**       |   Yes   | The version of the addon. It is a broker service identifier.  |
+|          **id**         |   Yes   | The broker service identifier.  |
+|     **description**     |   Yes   | The short description of the service. |
+|     **displayName**     |   Yes   | The display name of the addon.    |
+|         **tags**        |   No  | Keywords describing the provided service, separated by commas.     |
+|       **bindable**      |   No  | The field that specifies whether you can bind a given addon. |
+| **providerDisplayName** |   No  | The name of the upstream entity providing the actual service.  |
+|   **longDescription**   |   No  | The long description of the service.     |
+|   **documentationURL**  |   No  | The link to the documentation page for the service.        |
+|      **supportURL**     |   No  | The link to the support page for the service.     |
+|       **imageURL**      |   No  | The URL to an image. You must provide the image in the `SVG` format.          |
+|       **labels**        |   No  | Key-value pairs that help you to organize your project. Use labels to indicate different elements, such as Namespaces, services, or teams.   |
+| **bindingsRetrievable** |   No  | The field that specifies whether fetching a ServiceBinding using a GET request on the resource's endpoint is supported for all plans. The default value is `false`.   |
+|   **planUpdatable**     |   No  |  The field that specifies whether instances of this service can be updated to a different plan. The default value is `false`  |
+|       **requires**      |   No  | The list of permissions the user must grant to the instances of this service. |
+| **provisionOnlyOnce**   |   No  | The field that specifies whether the addon can be provisioned only once in a given Namespace. The default value is `false`. |
+
+> **NOTE**: The **provisionOnlyOnce** and **local** keys are reserved and cannot be added to the **labels** entry, since the Helm Broker overrides them at runtime. The Helm Broker always adds the `local:true` label and it adds the `provisionOnlyOnce:true` label only if **provisionOnlyOnce** is set to `true`.
```

Comment:
```suggestion
> **NOTE**: The **provisionOnlyOnce** and **local** keys are reserved and cannot be added to the **labels** entry, since Helm Broker overrides them at runtime. Helm Broker always adds the `local:true` label and it adds the `provisionOnlyOnce:true` label only if **provisionOnlyOnce** is set to `true`.
```